packages/mds-ingest-service/repository/entities/device-entity.ts

```diff
@@ -17,10 +17,9 @@
 import { IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
 import { ACCESSIBILITY_OPTION, MODALITY, Nullable, PROPULSION_TYPE, UUID, VEHICLE_TYPE } from '@mds-core/mds-types'
 import { Column, Entity } from 'typeorm'
-import { MigratedEntity } from '../mixins/migrated-entity'
 
 @Entity('devices')
-export class DeviceEntity extends MigratedEntity(IdentityColumn(RecordedColumn(class {}))) {
+export class DeviceEntity extends IdentityColumn(RecordedColumn(class {})) {
```

Comment:
I think you need to add `MigratedEntity` back onto all of these things now that we're only dropping the indices and not the columns themselves

Reply:
Note: `MigratedEntity` may need to be updated to not inherently add the indices

Reply:
Oh good catch, thanks.